### PR TITLE
xeadm improvements including bionic support

### PR DESCRIPTION
**Pull request by yamt**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change

/kind bug

> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

**What this PR does / why we need it**:

`kubeedge init` succeeded with these changes on ubuntu bionic.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: keadm/app/cmd/util/ubuntuinstaller.go [hunk 1]
```diff
@@ -294,6 +296,14 @@ func (u *UbuntuOS) addK8SRepositoryAndUpdate() error {
 		return fmt.Errorf("ubuntu dist version not available")
 	}
 	fmt.Println("Ubuntu distribution version is", distVersion)
+	distVersionForSuite := distVersion
+	if distVersion == "bionic" {
```
Comment (samy2019) [inline keadm/app/cmd/util/ubuntuinstaller.go]: Define "bionic" and "xenial" as constants and use it here.

Comment (yamt) [reply keadm/app/cmd/util/ubuntuinstaller.go]: done

**Comment by samy2019** [comment]
@yamt , Changes look go to me. Please squash the commits to one commit.

**Comment by naveensriram** [comment]
@yamt , could you please squash the commits to single commit and submit...many are looking for Bionic Ubuntu support...

**Comment by yamt** [comment]
@samy2019 @naveensriram why to squash? they are logically independent changes.

**Comment by samy2019** [comment]
@yamt , For Maintenance and tracking of master branch, we have to make sure that there is a single PR with single commit <-mapped-> against an Issue. 
Also the commits you have added are good for history in your branch, But in master it will create concerns. 
@sids-b , being a maintainer of KubeEdge, do add you views to explain @yamt .

**Comment by sids-b** [comment]
Thanks @samy2019 , .... @yamt we try to avoid small commits like fix typo if they can be merged with other. I will squash and merge this one. Thanks @yamt

**Pull request merged**